beacon_node/network/src/beacon_processor/worker/gossip_methods.rs

```diff
@@ -651,24 +651,98 @@ impl<T: BeaconChainTypes> Worker<T> {
     #[allow(clippy::too_many_arguments)]
     pub async fn process_gossip_blob(
         self,
-        _message_id: MessageId,
+        message_id: MessageId,
         peer_id: PeerId,
         _peer_client: Client,
         blob_index: u64,
         signed_blob: SignedBlobSidecar<T::EthSpec>,
         _seen_duration: Duration,
     ) {
+        let slot = signed_blob.message.slot;
+        let root = signed_blob.message.block_root;
+        let index = signed_blob.message.index;
         match self
             .chain
             .verify_blob_sidecar_for_gossip(signed_blob, blob_index)
         {
             Ok(gossip_verified_blob) => {
+                info!(
```

Comment:
The equivalent log is `debug` for blocks, maybe we should downgrade this one too cause we can get it up to 4 times per slot